lib/src/parse/stylesheet.dart

```diff
@@ -57,6 +57,20 @@ abstract class StylesheetParser extends Parser {
   /// Whether the parser is currently within a parenthesized expression.
   var _inParentheses = false;
 
+  SilentComment _lastSilentComment;
+
+  set lastSilentComment(SilentComment comment) => _lastSilentComment = comment;
```

Comment:
I'd just make the field public (and mark it `@protected`). Having a setter without a getter reads very strange to me.

Reply:
Done.